packages/sitecore-jss-angular/src/components/placeholder.component.spec.ts

```diff
@@ -1269,4 +1269,20 @@ describe('Placeholder Metadata: dynamic placeholder:', () => {
       expect(cleanedRenderedHTML).toEqual(expectedHTML);
     })
   );
+
+  fit(
```

Comment:
this should be "it(..." I guess?

Reply:
ah yes, I thought I cleaned up all spec changes. thanks for catching this